parts/windowsparams.t

```diff
@@ -67,4 +67,18 @@
         "description": "The version of Docker to be installed on Windows Nodes"
       },
       "type": "string"
+    },
+    "defaultContainerdRuntimeHandler": {
+      "defaultValue": "process",
+      "metadata": {
+        "description": "The containerd handler type (process isolated or hyperv)"
```

Comment:
Could we add allowed values for `defaultContainerdRuntimeHandler`

Reply:
May I ask the reason for your question? I cannot see any potential issues.

Reply:
The reason is somebody want to know the allowed values for `defaultContainerdRuntimeHandler`, just like swagger description. Does this make sense?
My comments left don't block you from merging, just suggestions.

Reply:
Allowed values were added in the description. `process` or `hyperv`

Reply:
BTW: This template is not exposed to customers but used internally.